x-pack/plugins/endpoint/public/applications/endpoint/store/alerts/mock_alert_result_list.ts

```diff
@@ -26,19 +26,24 @@ export const mockAlertResultList: (options?: {
   const alerts = [];
   for (let index = 0; index < actualCountToReturn; index++) {
     alerts.push({
-      '@timestamp': new Date(1542341895000).toString(),
+      '@timestamp': new Date(1542341895000),
+      id: 'xDUYMHABAJk0XnHd8rrd',
       agent: {
         id: 'ced9c68e-b94a-4d66-bb4c-6106514f0a2f',
         version: '3.0.0',
       },
       event: {
+        id: '2f1c0928-3876-4e11-acbb-9199257c7b1c',
```

Comment:
Added `event.id` for tie-breaking sorts. It's not recommended to use Elastic's `_id` for tie-breaking.